### PR TITLE
Update Readme for Govee devices

### PR DESCRIPTION
**Pull request by Thrilleratplay**
@Ernst79 I wasn't sure if listing the devices was done by shared `device_type` or by individual device packaging.  Let me know if you want any changes.

### CODE REVIEW

**Comment by Ernst79** [comment]
Looks perfect, just one question. Do H5101/H5102 need active scan enabled? You didn't mention it for that one.

File: README.md [hunk 1]
```diff
@@ -79,7 +79,15 @@ This integration supports **Xiaomi MiBeacon, Qingping, ATC, Xiaomi Scale, Kegtro
...
+| **H5074** |**Govee H5074 Thermometer Hygrometer**<br/><br/>Square body, no screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5074](/pictures/Govee_H5074.jpg)|
+| **H5075** |**Govee H5075 Thermometer Hygrometer**<br/><br/>Rounded square body, LCD screen.  Broadcasts temperature, humidity and battery level.  Govee devices only send data with the option [active_scan](#active_scan) set to `True`, so make sure you change this setting, as the default is `False`|![H5075](/pictures/Govee_H5075.jpg)|
+| **H5101, H5102** |**Govee H5101/Govee H5102 Thermometer Hygrometer**<br/><br/>Rounded square body, LCD screen.  Broadcasts temperature, humidity and battery level.|![H5101/H5102](/pictures/Govee_H5101_H5102.jpg)|
```
Comment (Ernst79) [inline README.md:86]: No active scan for H5101/H5102?

Comment (Thrilleratplay) [reply README.md:86]: It is required.  I'm updating it.  Thanks for catching that.

**Comment by Ernst79** [comment]
ok, thanks. I'll make a beta release tomorrow

**Pull request merged**